specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-04-01-preview/cosmos-db.json

```diff
@@ -8861,6 +9125,322 @@
           "description": "The latest restorable timestamp for a resource."
         }
       }
+    },
+    "DataTransferClusterResource": {
```

Comment:
>DataTransferClusterResource [](start = 5, length = 27)

This will be called ServiceResource. There is a concept of discriminator in swagger, you can use that to differentiate between various ServiceTypes. 

Refer this: [https://swagger.io/specification/?sbsearch=discriminator#discriminator-object](https://swagger.io/specification/?sbsearch=discriminator#discriminator-object)